tools/eubfr-cli/commands/delete.js

```diff
@@ -0,0 +1,114 @@
+const url = require('url');
+const path = require('path');
+
+const aws4 = require('aws4');
+const dotenv = require('dotenv');
+const elasticsearch = require('elasticsearch');
+const request = require('request-promise-native');
+
+const getServiceLocation = require('../lib/getServiceLocation');
+
+dotenv.config({
+  path: path.resolve(getServiceLocation('storage-deleter'), '.env'),
+});
+
+/**
+ * Deletes files from S3 for a given producer.
+ *
+ * @param {Array} files
+ *   List of files to delete.
+ * @param {Array} credentials
+ *   List of credentials for the producers.
+ */
+const deleteCommand = async ({ files, credentials }) => {
```

Comment:
Technically, this `async` is not needed, the function doesn't directly uses `await` and already returns a Promise but it's okay to keep it, it's like a safety net that ensures we always return a Promise :)